scss/_normalize.scss

```diff
@@ -0,0 +1,350 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Also about Normalize. Just sharing with you for future purposes. There is an npm package that has normalize. So if you're building a project that already has a build stem setup, it would be best to use the package, that way your project only has files that were coded by your team (Unless in a very rare circumstance you want to make some changes to normalize)